diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/WhiteSpaceRule.kt

```diff
@@ -2,33 +2,60 @@ package org.cqfn.diktat.ruleset.rules
 
 import com.pinterest.ktlint.core.KtLint
 import com.pinterest.ktlint.core.Rule
+import com.pinterest.ktlint.core.ast.ElementType.ARROW
```

Comment:
I am starting to think that wildcard are not so bad idea...

Reply:
Maybe, we should support exceptions to several wildcards through configuration, like
```
   allowedWildcards="com.pinterest.ktlint.core.ast.*,..."
```

Reply:
Created #185